components/image/image.css

```diff
@@ -0,0 +1,15 @@
+.image-responsive {
+	width: 100%;
+	height: auto;
+}
+
+.image-round {
+	width: 25rem;
+	height: 25rem;
+	border-radius: 50%;
+	object-fit: cover;
+}
+
+.mt-4 {
+	margin-top: 4rem;
+}
```

Comment:
You can add this utility class in another separate file. Might help you in long run.